src/app/components/VerifyingIfAnswerIsCorrect.js

```diff
@@ -0,0 +1,6 @@
+
+export default function VerifyingAnswerIsCorrect(userAnswer, rightAnswer) {
+  const userAnswerIsCorrect = { correct: "" };
+  userAnswer === rightAnswer ? userAnswerIsCorrect.correct = true : userAnswerIsCorrect.correct = false;
+  return userAnswerIsCorrect
```

Comment:
Does it have to return Object? 
If it's only have to compare 2 answers, why not write it just like this:
`return userAnswer===rightAnswer;`